sagemaker-spark/parameterize-spark-config-pysparkprocessor-pipeline/parameterize-spark-config-pysparkprocessor-pipeline.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<p>I would use our designated notebook examples bucket abalone dataset copy at s3://sagemaker-sample-files/datasets/tabular/uci_abalone/ - this is because we are in charge of this bucket and will ensure it is not broken, but cannot guarantee that for other buckets. Let me know if you want me to upload the preprocessed copy or you can preprocess it here.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
> I would use our designated notebook examples bucket abalone dataset copy at s3://sagemaker-sample-files/datasets/tabular/uci_abalone/ - this is because we are in charge of this bucket and will ensure it is not broken, but cannot guarantee that for other buckets. Let me know if you want me to upload the preprocessed copy or you can preprocess it here.
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3685/discussion/)_

I updated abalone dataset source to s3://sagemaker-sample-files/datasets/tabular/uci_abalone/abalone.csv